rebuild_fuzzer_corpus.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import fontTools.ttLib.ttFont as ttFont
+import fontTools.ttLib.tables.C_O_L_R_ as C_O_L_R_
+
+SRC_FONT = "fonts/noto-glyf_colr_1.ttf"
+SAMPLES_FONT = "fonts/more_samples-glyf_colr_1.ttf"
+NUM_TOP_BUSY_PAINTS = 90
+
+ttfont = ttFont.TTFont(SRC_FONT)
+
+glyphs_paintcount = []
+for g in ttfont["COLR"].table.BaseGlyphList.BaseGlyphPaintRecord:
+    counter = 0
+
+    def traversePaint(paint):
+        global counter
+        counter += 1
+
+    g.Paint.traverse(ttfont["COLR"].table, traversePaint)
+    glyphs_paintcount.append((ttfont.getGlyphID(g.BaseGlyph), counter))
+
+glyphs_paintcount = sorted(glyphs_paintcount, key=lambda x: x[1], reverse=True)
+
+high_paint_count_glyphs = [g[1] for g in glyphs_paintcount[0:NUM_TOP_BUSY_PAINTS]]
```

Comment:
I believe you meant to get the first item of the tuples which is the glyph index, not the second one which is the counter:

```suggestion
high_paint_count_glyphs = [g[0] for g in glyphs_paintcount[0:NUM_TOP_BUSY_PAINTS]]
```

maybe using a namedtuple (accessing members by their name instead of position) would make things clearer

Reply:
Using a namedtuple now, hope that looks better. Good catch regarding the index... 🤐 Updated PR for freetype2-testing here https://github.com/freetype/freetype2-testing/pull/109/files - thanks for pointing that out. pyftsubset did not complain and glyph id's seemed reasonable - but they were indeed nicely ordered paint counts...